src/Template/Pages/Modules/index.twig

```diff
@@ -20,7 +20,7 @@
 
             <div class="list-objects">
                 {% if objects %}
-                    {% element 'Modules/index_table_header' %}
+                    {% element 'Modules/index_table_header' { 'object': objects[0] } %}
```

Comment:
[super minor] maybe `firstObject` or `refObject` instead of `object` here and in `Modules/index_table_header.twig` to avoid confusion with `object/objects` vars else where